observation_portal/observations/models.py

```diff
@@ -227,7 +244,6 @@ def as_dict(self):
         return import_string(settings.AS_DICT['observations']['ConfigurationStatus'])(self)
 
     class Meta:
-        unique_together = ('configuration', 'observation')
```

Comment:
I'm not sure why we had this constraint initially - I don't see any reason that we need it, and it is impossible if we want to allow multiple configuration statuses to point to the same configuration (which is how nodding is implemented here).

Reply:
Yeah, that makes sense.